rcl/test/rcl/test_message_info.cpp

```diff
@@ -0,0 +1,336 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
+// Copyright 2020 Robert Bosch GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <string>
+#include <thread>
+
+#include "rcl/subscription.h"
+
+#include "rcl/rcl.h"
+#include "rcl/graph.h"
```

Comment:
Do you need all of these includes?

Reply:
Yes, I'm using graph.h to check whether the service is up. It seemed more appropriate than having another of those 1s delays in there to make sure the connection is established ;-) If there's a better way, please let me know!